test/node/ol/render/Feature.test.js

```diff
@@ -0,0 +1,247 @@
+          [0, 0],
+          [4, 5],
+        ],
+        [
+          [0, 0],
+          [4, 5],
+        ],
+      ]);
+      const renderFeature = new RenderFeature(
+        geometry.getType(),
+        geometry.getFlatCoordinates().slice(),
+        geometry.getEnds().slice()
+      );
+      const converted = toGeometry(renderFeature);
+      expect(converted).to.be.a(MultiLineString);
+      expect(converted.getFlatCoordinates()).to.eql(
+        geometry.getFlatCoordinates()
+      );
+      expect(converted.getEnds()).to.eql(geometry.getEnds());
+      expect(converted.getProperties()).to.eql({});
+    });
+    it('creates a Polygon', function () {
+      const geometry = new Polygon([
+        [
+          [0, 0],
+          [5, 0],
+          [5, 5],
+          [0, 0],
+        ],
+        [
+          [1, 1],
+          [4, 4],
+          [4, 1],
+          [1, 1],
+        ],
+      ]);
+      const renderFeature = new RenderFeature(
+        geometry.getType(),
+        geometry.getFlatCoordinates().slice(),
+        geometry.getEnds().slice()
+      );
+      const converted = toGeometry(renderFeature);
+      expect(converted).to.be.a(Polygon);
+      expect(converted.getFlatCoordinates()).to.eql(
+        geometry.getFlatCoordinates()
+      );
+      expect(converted.getEnds()).to.eql(geometry.getEnds());
+      expect(converted.getProperties()).to.eql({});
+    });
+    it('creates a MultiPolygon from oriented polygon rings', function () {
+      const geometry = new MultiPolygon([
+        [
+          [
+            [0, 0],
+            [5, 0],
+            [5, 5],
+            [0, 0],
+          ],
+          [
+            [1, 1],
+            [4, 4],
+            [4, 1],
+            [1, 1],
+          ],
+        ],
+        [
+          [
+            [-0, -0],
+            [-5, -0],
+            [-5, -5],
+            [-0, -0],
+          ],
+        ],
+      ]);
+      const renderFeature = new RenderFeature(
+        GeometryType.POLYGON,
+        geometry.getFlatCoordinates().slice(),
+        geometry.getEndss().flat(1)
+      );
+      const converted = toGeometry(renderFeature);
+      expect(converted).to.be.a(MultiPolygon);
+      expect(converted.getFlatCoordinates()).to.eql(
+        geometry.getFlatCoordinates()
+      );
+      expect(converted.getEndss()).to.eql(geometry.getEndss());
+      expect(converted.getProperties()).to.eql({});
+    });
+    it('creates a MultiPolygon', function () {
+      const geometry = new MultiPolygon([
+        [
+          [
+            [0, 0],
+            [5, 0],
+            [5, 5],
+            [0, 0],
+          ],
+          [
+            [1, 1],
+            [4, 1],
+            [4, 4],
+            [1, 1],
+          ],
+        ],
+        [
+          [
+            [-0, -0],
+            [-5, -0],
+            [-5, -5],
+            [-0, -0],
+          ],
+        ],
+      ]);
+      const renderFeature = new RenderFeature(
+        geometry.getType(),
```

Comment:
There is no MULTI_POLYGON type for a render feature. You can remove this test case.